src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceInvocationHandler.java

```diff
@@ -260,6 +261,12 @@ private static FullHttpResponse convertResponse(Response coyoteRes, ByteBuf cont
                 HttpVersion.HTTP_1_1, HttpResponseStatus.valueOf(coyoteRes.getStatus()), content);
 
         final HttpHeaders headers = res.headers();
+
+        final String contentType = coyoteRes.getContentType();
+        if (contentType != null && !contentType.isEmpty()) {
+            headers.add(HttpHeaderNames.CONTENT_TYPE.toString(), contentType);
```

Comment:
- You don't need `toString()`.
- How about using `set()`, given that the headers do not have multiple `Content-Type` headers?